autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/sync/persister/CredentialsLastModifiedWinsStrategy.kt

```diff
@@ -71,12 +77,12 @@ class CredentialsLastModifiedWinsStrategy(
         localCredential: LoginCredentials,
         remoteEntry: CredentialsSyncEntryResponse,
         clientModifiedSince: String,
-    ) {
+    ): Boolean {
```

Comment:
I’ve added a description to the method, I prefer not to rename it

Reply:
Sorry for being a pain with this detail, but I don't like the method returning a `Boolean` that represents an implementation detail of the method.

The I read:
```
if (processExistingEntry(localCredential, entry, clientModifiedSince)) {
    hasDataChangedWhileSyncing = true
}
```
and I don't see why when `processExistingEntry` is `true` means `hasDataChangedWhileSyncing` is `true`. So it's hard to follow.

I see 2 options:
a) Return `SyncMergeResult` instead (this would be my pref)
b) change the definition of that `Boolean` to be `true if remote entry wins, false otherwise`.

Reply:
discussed offline, we are refactoring a little bit to clean it up